src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public enum RoleName {
+        ADMIN,
+        USER
+    }
```

Comment:
It is like inner class. Should be after public methods.
additionally, I don`t see equals and hashcode. check checklist № 4